tests/sentry/migrations/test_0913_split_discover_dataset_dashboards_self_hosted.py

```diff
@@ -10,6 +12,7 @@
 from sentry.utils.samples import load_data
 
 
+@pytest.mark.skip("Skipping test b/c it breaks tests for 0068_migrate_anomaly_detection_alerts")
```

Comment:
do you know what's causing this to break our test? also just want to make sure you've reached out to the team that owns this area so they're aware of the issue (or maybe dev-infra?)